### ISSUE TITLE
Links are not enabled - PDFJS undefined

### ISSUE THREAD

**Issue by zeecorleone** [mozilla/pdf.js#14786]
I have a pdf file that contains some links, which point to some specific page within the same pdf document. They are not enabled/clickable when viewing the pdf file. How can it be enabled? 
I tried to search and found that it could be enabled by "enabling annotation layer". But all examples I found are using `PDFJS` object, which I believe is not available in versions 2+. 
For example a part of the code I'm trying uses:

` PDFJS.Annotation.fromData(data);`

and it breaks, as `PDFJS` doesn't exist.
 
![image](https://user-images.githubusercontent.com/9975195/163670466-ebf1a99a-8517-4fb3-bdc8-9993823846d9.png)

To sum it up, I'm trying to enable the links, making them clickable. How can I do this? any help/guideline will be appreciated.

**Configuration:**
- Web browser and its version: Chrome
- Operating system and its version: windows 10
- PDF.js version: 2.6.347'

**Comment by Snuffleupagus** [comment]
> Web browser and its version: Chrome

*For future reference:* Please remember to always specify the version *number* as well.

> PDF.js version: 2.6.347

That version is no longer supported, please find the latest releases at https://mozilla.github.io/pdf.js/getting_started/#download

---

I'd suggest taking a look at the `viewer component` examples, see https://github.com/mozilla/pdf.js/tree/master/examples/components, since the contain the necessary components for building a (basic) viewer. Among other things they support text-selection, annotation rendering, and form filling by default.
